components/chassis.py

```diff
@@ -221,19 +226,12 @@ def drive_local(self, x, y, z):
     def _drive(self, chassis_speeds):
         self.desired_states = self.kinematics.toSwerveModuleStates(chassis_speeds)
         for state, module in zip(self.desired_states, self.modules):
-            # new_state = SwerveModuleState.optimize(state, module.get_rotation())
-            module.set(state)
+            new_state = SwerveModuleState.optimize(state, module.get_rotation())
+            module.set(new_state)
```

Comment:
nit: make it easier to comment out the optimize call
```suggestion
            state = SwerveModuleState.optimize(state, module.get_rotation())
            module.set(state)
```